main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -64,7 +64,7 @@ object Safety {
     */
   private def visitDefs(root: Root)(implicit flix: Flix): List[CompilationMessage] = {
     root.defs.flatMap {
-      case (_, defn) => visitDef(defn, root)
+      case (_, defn) => visitDef(defn, root) ::: visitTestEntryPoint(defn, root)
```

Comment:
Is there a reason this is not part of visitDef`?

Reply:
I thought it was clumsy in that function, but I experimented with it being there. I can put it back there if you want.